src/util.cpp

```diff
@@ -722,10 +722,10 @@ fs::path GetDefaultDataDir()
         pathRet = fs::path(pszHome);
 #ifdef MAC_OSX
     // Mac
-    return pathRet / "Library/Application Support/Bitcoin";
+    return pathRet / "Library/Application Support/Elements";
 #else
     // Unix
-    return pathRet / ".bitcoin";
+    return pathRet / ".elements";
```

Comment:
Do we want to do this? It's always been `.bitcoin`, except for Liquid public release which was `.liquid`.